api_bitrise.py

```diff
@@ -0,0 +1,99 @@
+import sys
+import os
+
+import pandas as pd
+from datetime import timezone
+from sqlalchemy import func
+
+from lib.bitrise_conn import BitriseAPIClient
+from utils.datetime_utils import DatetimeUtils as dt
+from database import (
+    Database,
+    ReportBitriseBuildsCount
+)
+
+
+class Bitrise:
+
+    def __init__(self):
+        try:
+            BITRISE_HOST = os.environ['BITRISE_HOST']
+            self.client = BitriseAPIClient(BITRISE_HOST)
+            self.client.BITRISE_APP_SLUG = os.environ['BITRISE_APP_SLUG']
+            self.client.token = os.environ['BITRISE_TOKEN']
+        except KeyError:
+            print("ERROR: Missing bitrise env var")
+            sys.exit(1)
+
+    # API: Builds
+    def builds(self, past_date_timestamp):
+        return self.client.get_builds(self.client.BITRISE_APP_SLUG, past_date_timestamp) # noqa
+
+    def get_builds_range_date(self, after):
```

Comment:
NIT: using get_ can be confusing because it might appear your using an outdated getter setter pattern.  If the value returned is builds range date then the function can simply be called `builds_range_date`.  Also, what is this data exactly its a bit unclear from the function name.  Is it a date range?